src/Pages/ArtCollection.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+
+class ArtCollection extends Component {
```

Comment:
?

Reply:
deleted now, it was a component i made that i never ended up using.